homework10/Button Squares/Button Squares/ViewController.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  ViewController.swift
+//  Button Squares
+//
+//  Created by Ernest Mihasenko on 1.04.22.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+
+    @IBOutlet weak var space: UIView!
+    @IBOutlet weak var secondSquare: UILabel!
+    @IBOutlet weak var firstSquare: UILabel!
+    @IBOutlet weak var showSquaresButton: UIButton!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view.
+        
+        firstSquare.isHidden = true
+        secondSquare.isHidden = true
+    }
+
+    @IBAction func pressButton(_ sender: UIButton) {
+        firstSquare.isHidden = false
+        secondSquare.isHidden = false
+      
+        firstSquare.frame.origin = CGPoint(x: Int.random(in: Int(space.frame.minX)..<Int(space.frame.maxX - 90)), y: Int.random(in: Int(space.frame.minY)..<Int(space.frame.maxY - 90)))
+     
+       secondSquare.frame.origin = CGPoint(x: Int.random(in: Int(space.frame.minX)..<Int(space.frame.maxX - 90)), y: Int.random(in: Int(space.frame.minY)..<Int(space.frame.maxY - 90)))
```

Comment:
There's no need to have these lines here because you have the same lines in the `repeat while`. So you can remove them